content/projects/gsoc/2022/application.adoc

```diff
@@ -0,0 +1,260 @@
+---
+layout: project
+title: "Jenkins GSoC Application 2022"
+description: "This page is Jenkins application form for Google Summer of Code"
+tags:
+- gsoc
+project: gsoc
+opengraph:
+    image: /images/gsoc/opengraph.png
+---
+
+//https://docs.google.com/document/d/1gVO2XT_7m8kf5aJJS00G05f5zBBf1eaFZPcoWl8W0m0/edit#heading=h.pj9miykzw86z
+
+{nbsp} +
+{nbsp} +
+
+
+**Organization Name**
+
+Jenkins
+
+{nbsp} +
+{nbsp} +
+ 
+
+**Is this organization an established Open Source project or organization that releases code under an OSI approved license ? An OSI License is required for program participation.**
+
+Yes
+
+{nbsp} +
+{nbsp} +
+
+
+**Has your organization participated in Google Summer of Code before?**
+
+Yes
+
+{nbsp} +
+{nbsp} +
+
+
+**Please select all years in which your organization participated prior to 2022.**
+
+2021, 2020, 2019, 2018, 2016
+
+{nbsp} +
+{nbsp} +
+
+
+**Agreements and Rules**
+
+_link:https://summerofcode.withgoogle.com/terms/org[Click the link] to review the organization agreement to continue._
+
+>> Accept
+
+{nbsp} +
+{nbsp} +
+
+
+=== Organization Profile
+
+* **Website url:** https://jenkins.io/
+* **Organization logo**
+
+<logo comes here>
+
+* **Tagline:** Open-source automation server for building great things at any scale
+
+* **Primary Open Source License:** MIT License is used for most of the components.
+
+* **What year was your project started:** 2006
+
+* **Link to your source code location:** 
+** https://github.com/jenkinsci/, 
+** https://github.com/jenkins-zh/,
+** https://github.com/jenkins-infra/ 
+
+
+{nbsp} +
+{nbsp} +
+
+
+=== Organization Categories 
+_Select which categories fit your organization best. You may select up to 2 categories. This helps GSoC contributors filter the large list of organizations by their interests._
+_Select at least 1_
+
+* Data (databases, analytics, visualization, AI/ML, etc) 
+* **Development tools** (version control systems, CICD tools, text editors, issue managers, q/a tools, etc) 
+* End user applications 
+* **Infrastructure and cloud** (hardware, software defined infrastructure, cloud native tooling, orchestration and automation, etc) 
```

Comment:
Applications for previous users placed us under "Programming languages" rather than "Infrastructure and cloud"
```suggestion
* Infrastructure and cloud (hardware, software defined infrastructure, cloud native tooling, orchestration and automation, etc) 
```